packages/lesswrong/components/review/LatestReview.tsx

```diff
@@ -37,14 +38,30 @@ const LatestReview = ({classes}) => {
     limit: 1
   });
 
+  const { hover, anchorEl, eventHandlers } = useHover({
+    pageElementContext: "FrontpageReviewWidget",
+    pageElementSubContext: "LatestReviews",
+  })
+
   if (!commentResults?.length) return null
   const comment = commentResults[0]
   if (!comment.post) return null
 
-  return <ErrorBoundary><div className={classes.root}>
-    <LWTooltip tooltip={false} title={<PostsPreviewTooltipSingleWithComment postId={comment.postId} commentId={comment._id}/>}>
-      <Link to={commentGetPageUrlFromIds({postId: comment.postId, commentId: comment._id, postSlug: comment.post.slug})} className={classes.lastReview}>Latest Review: <span className={classes.title}>{comment.post.title}</span></Link>
-    </LWTooltip>
+  return <ErrorBoundary><div className={classes.root} {...eventHandlers} >
```

Comment:
I rearranged this component so that you could move your mouse over the LatestReview hover-preview, and click buttons on it (so people could read the review and then decide to change their vote on the post)